jenkinsfiles/acm-qe.Jenkinsfile

```diff
@@ -154,10 +162,17 @@ pipeline {
                     if (params.TEST_TAGS == '@post-release') {
                         DOWNSTREAM = "--downstream false"
                     }
+
+                    // Use the '@pre-upgrade' tag to update the Submariner catalog
+                    if (params.TEST_TAGS == '@pre-upgrade') {
+                        DEPLOY_OR_UPDATE = "--subm-catalog-update"
```

Comment:
Could you change the name of the variable to - FLOW?
It will be much more convenient.